event/src/unvalidated/signed/mod.rs

```diff
@@ -192,6 +197,34 @@ impl Envelope {
                 })
         })
     }
+
+    /// Get the signature
+    pub fn get_signature(&self) -> &Bytes {
+        &self.signatures[0].signature
```

Comment:
This will panic if the vec is empty. Since we use a builder, we shouldn't have to worry about it right now, but maybe in the future we can add a custom field deserializer to error if it's ever empty so we are guaranteed to have one at least one afterward.

But more to the point, maybe we can make this `signatures(&self) ->&[Signature]` and let the caller deal with it.